Git-commit: 685ec4c0f27c6fc0364e2dc6fbeb9257175f1e9c
From: Geert Uytterhoeven <geert+renesas@glider.be>
Date: Wed, 31 Mar 2021 10:15:19 +0200
Subject: [PATCH] reset: RESET_INTEL_GW should depend on X86
References: bsn#19
Patch-mainline: v5.10.52

[ Upstream commit 6ab9d6219f86f0db916105444813aafce626a2f4 ]

The Intel Gateway reset controller is only present on Intel Gateway
platforms.  Hence add a dependency on X86, to prevent asking the user
about this driver when configuring a kernel without Intel Gateway
support.

Fixes: c9aef213e38cde27 ("reset: intel: Add system reset controller driver")
Signed-off-by: Geert Uytterhoeven <geert+renesas@glider.be>
Signed-off-by: Philipp Zabel <p.zabel@pengutronix.de>
Signed-off-by: Sasha Levin <sashal@kernel.org>
Signed-off-by: Kai Liu <kai.liu@suse.com>
---
 drivers/reset/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/reset/Kconfig b/drivers/reset/Kconfig
index b1d7369218e8..147543ad303f 100644
--- a/drivers/reset/Kconfig
+++ b/drivers/reset/Kconfig
@@ -76,6 +76,7 @@ config RESET_IMX7
 
 config RESET_INTEL_GW
 	bool "Intel Reset Controller Driver"
+	depends on X86 || COMPILE_TEST
 	depends on OF && HAS_IOMEM
 	select REGMAP_MMIO
 	help
-- 
2.31.1

